src/librustdoc/html/render/print_item.rs

```diff
@@ -284,16 +286,14 @@ fn item_module(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item, items: &[cl
 
             clean::ImportItem(ref import) => {
                 let (stab, stab_tags) = if let Some(import_def_id) = import.source.did {
-                    let import_attrs = Box::new(clean::Attributes::from_ast(
-                        cx.tcx().sess().diagnostic(),
-                        cx.tcx().get_attrs(import_def_id),
-                        None,
-                    ));
+                    let ast_attrs = cx.tcx().get_attrs(import_def_id);
+                    let import_attrs = Box::new(clean::Attributes::from_ast(ast_attrs, None));
 
                     // Just need an item with the correct def_id and attrs
                     let import_item = clean::Item {
                         def_id: import_def_id,
                         attrs: import_attrs,
+                        cfg: ast_attrs.cfg(cx.tcx().sess().diagnostic()),
```

Comment:
The function call is unnecessary.
```suggestion
                        cfg: ast_attrs.cfg(cx.tcx().sess.diagnostic()),
```

Reply:
It's done. What are is the difference between calling the function and the field ? When should I use one over an other ?

Reply:
There's no difference. You should always use the field unless you're in a generic context where you don't know if you have a TyCtxt or not.

Reply:
Ok thanks